bootstrap/shell/test.go

```diff
@@ -13,7 +13,7 @@ import (
 func NewTestShell(t *testing.T) *Shell {
 	sh, err := New()
 	if err != nil {
-		t.Fatal(err)
+		t.Fatalf("shell.New() error = %v", err)
```

Comment:
What's your perspective on converting these to be black box test? From a cursory glance, I think for this file at least, it should be straightforward.

Reply:
I'm in favour of decreasing how aware the tests are of how the code operates (something like "test what it does, not how it does it" - there's a good quote somewhere I can't seem to find...). Probably a higher priority would be increasing coverage on other critical agent pieces.